Sources/PaymentButtons/PayPalPayLaterButton.swift

```diff
@@ -38,67 +38,61 @@ public final class PayPalPayLaterButton: PaymentButton {
-        private var action: () -> Void = { }
-        
-        /// Initialize a PayPalPayLaterButton
-        /// - Parameters:
-        ///   - insets: Edge insets of the button, defining the spacing of the button's edges relative to its content.
-        ///   - color: Color of the button. Default to gold if not provided.
-        ///   - edges: Edges of the button. Default to softEdges if not provided.
-        ///   - size: Size of the button. Default to standard if not provided.
-        public init(
-            insets: NSDirectionalEdgeInsets? = nil,
-            color: PayPalPayLaterButton.Color = .gold,
-            edges: PaymentButtonEdges = .softEdges,
-            size: PaymentButtonSize = .standard,
-            _ action: @escaping () -> Void = { }
-        ) {
-            self.button = PayPalPayLaterButton(
-                fundingSource: .payLater,
-                color: color.color,
-                edges: edges,
-                size: size,
-                insets: insets,
-                label: .payLater
-            )
-            self.action = action
-        }
+/// PayPalPayLaterButton for SwiftUI
+public struct PayPalPayLaterButtonView: UIViewRepresentable {
 
-        // MARK: - UIViewRepresentable methods
-        // TODO: Make unit test for UIVRepresentable methods: https://engineering.paypalcorp.com/jira/browse/DTNOR-623
-        
-        public func makeCoordinator() -> Coordinator {
-            Coordinator(action: action)
-        }
+    private let button: PayPalPayLaterButton
+    private var action: () -> Void = { }
 
-        public func makeUIView(context: Context) -> UIView {
-            let view = UIView()
-            view.addSubview(button)
-            
-            button.addTarget(context.coordinator, action: #selector(Coordinator.onAction(_:)), for: .touchUpInside)
+    /// Initialize a PayPalPayLaterButton
+    /// - Parameters:
+    ///   - insets: Edge insets of the button, defining the spacing of the button's edges relative to its content.
+    ///   - color: Color of the button. Default to gold if not provided.
+    ///   - edges: Edges of the button. Default to softEdges if not provided.
+    ///   - size: Size of the button. Default to standard if not provided.
+    public init(
+        insets: NSDirectionalEdgeInsets? = nil,
+        color: PayPalPayLaterButton.Color = .gold,
+        edges: PaymentButtonEdges = .rounded,
+        size: PaymentButtonSize = .standard,
+        _ action: @escaping () -> Void = { }
+    ) {
+        self.button = PayPalPayLaterButton(
+            fundingSource: .payLater,
+            color: color.color,
+            edges: edges,
+            size: size,
+            insets: insets,
+            label: .payLater
+        )
+        self.action = action
+    }
 
-            return view
-        }
+    // MARK: - UIViewRepresentable methods
+    // TODO: Make unit test for UIVRepresentable methods: https://engineering.paypalcorp.com/jira/browse/DTNOR-623
```

Comment:
Same here:
```suggestion
    // TODO: Make unit test for UIVRepresentable methods: ticket DTNOR-623
```

Reply:
This looks like a bad merge... I didn't type that and the link leads to a 404 error. Will remove